features/networking/ovn_ipsec.feature

```diff
@@ -198,70 +198,35 @@ Feature: OVNKubernetes IPsec related networking scenarios
   # @case_id OCP-40569
   @admin
   @destructive
-  @inactive
-  @network-ovnkubernetes @network-openshiftsdn @ipsec
+  @network-ovnkubernetes
   @4.12 @4.11
-  @vsphere-ipi @openstack-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi
```

Comment:
The removal of this line is intended or by mistake ?

Reply:
> The removal of this line is intended or by mistake ?

@liangxia Oops, put them back.